consensus/state.go

```diff
@@ -2013,6 +2018,8 @@ func (cs *State) handleCompleteProposal(blockHeight int64) {
 // Attempt to add the vote. if its a duplicate signature, dupeout the validator
 func (cs *State) tryAddVote(vote *types.Vote, peerID p2p.ID) (bool, error) {
 	added, err := cs.addVote(vote, peerID)
+
+	// NOTE: some of these errors are swallowed here
```

Comment:
Can you clarify? This comment looks a bit cryptic.

Reply:
We can improve the message, but not in this PR: I'd like it to stay a forward-port (so as close a possible to the original one)